### DISCUSSION TITLE
Welcome to govready-q Discussions!

### DISCUSSION THREAD

**Discussion by gregelin** [GovReady/govready-q#1163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SuperSushiRoll** [comment]
Greetings,

How would I go about creating an SSP? I don't know how to add applications, and I was wondering if there was an app out there that would let me tailor the questions relevant to my SSP, and use Govready to fill out and generate an SSP. I'm still pretty green to the field so I'm not sure I'm in the right place. I'm essentially looking for a program to streamline SSP creation, we have one (antiquated manually sustained word documents) and I'm looking to move to an automated system.

**Reply by cityinohio2019** [reply]
Hey SuperSushi!

The latest version of GovReady-Q has a set of files that ship with its install that can be used to create an SSP. These files can be copied and iterated on to fit your organization.

As far as adding new applications, you can enter the admin panel with Django and add you custom app sources from there. You can check out how to do that with this video:
https://vimeo.com/418916589

This page has a ton of videos detailing some of the ins and outs of navigating GovReady-Q including creating an original application that works with GovReady-Q
https://vimeo.com/govready

Let me know if that helps!
